bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -446,7 +438,7 @@ class PlotCanvasView extends Renderer.View
 
     for tool_view in tool_views
       level = tool_view.model.level
-      @levels[level][tool_view.model.id] = tool_view
+      @levels['tool'][tool_view.model.id] = tool_view
```

Comment:
Is this right? The normal case is 'tool' but the intent was that in some cases, some tools might want to specify their level differently.